### DISCUSSION TITLE
Unable To AccessMy Github Account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#38540] [Other Feature Feedback, Questions, & Ideas] [answered]
Hey guys, please I have a bit of an issue I need urgent help to solve;

Yesterday, I tried to log in with my password which I am certain of but got a 'weak or compromised password' which then asked me to reset the password with my 2FA. Unfortunately, the device on which I set up the 2FA was stolen and I am yet to deactivate it because of some project teams I joined.

I still have it accessible on my CLI but that would be a matter of time unless I fixed this. Please I need help to be able to continue my work on time.

**Comment by airtower-luna** [comment, marked as answer]
See how to [Recover an account with 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). If none of that works for you, you can try to [contact GitHub support](https://support.github.com/request), but as noted in the documentation they may not be able to help you without 2FA. Good luck!

**Reply by ghost** [reply]
The problem here is my 2FA, I should be able to recover with other methods right? 

Also because my password is correct.

I am not able to recover the 2FA code on my device. I opened a ticker on the support page but no response to it yet.

**Reply by airtower-luna** [reply]
You lost your 2FA, so you need to use one of the recovery methods for that case, as described in the documentation I linked above.

**Reply by ghost** [reply]
2FA seems to be the first resort then the recovery codes which I can not find. I tried my access token from my gh CLI but that does not work either.

Please what can I do?

**Reply by airtower-luna** [reply]
There are a few more options described in that article. If you don't have any of the required things, your only option is to wait **if** support can help you. Us other users definitely can't.

**Reply by ghost** [reply]
Okay, I understand. Thank you very much @airtower-luna

**Comment by ghost** [comment]
The problem here is my 2FA, I should be able to recover with other methods right?

Also because my password is correct.

I am not able to recover the 2FA code on my device. I opened a ticker on the support page but no response to it yet.
